test/e2e/scenario/recorder/recorder.scenario.ts

```diff
@@ -658,6 +661,65 @@ describe('recorder', () => {
         expect(segment.records.slice(3).every((record) => record.type !== RecordType.FullSnapshot)).toBe(true)
       })
   })
+
+  describe('frustration records', () => {
+    createTest('should detect a dead click and match it to mouse interaction record')
+      .withRum({ trackFrustrations: true })
+      .withRumInit(initRumAndStartRecording)
+      .withSetup(bundleSetup)
+      .run(async ({ serverEvents }) => {
+        await browserExecute(() => document.documentElement.outerHTML)
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** Is it actually useful or is it a debug leftover?